src/js/themes/base.d.ts

```diff
@@ -625,6 +625,11 @@ export interface ThemeType {
         background?: BackgroundType;
       };
       pad?: PadType;
+      units?: {
+        gap?: GapType;
+        align?: string;
```

Comment:
Should this be `AlignType`?

Reply:
Somewhere else was just doing string but that would be much better

Reply:
Done

Reply:
Feel free to fix the other place as well.

Reply:
Can we merge this without updating the other usages, especially since I didn't have to specify it here with the container change? I'm actually supposed to be on FTO since Wednesday.

Reply:
No rush. we rather have all the relevant fixes in one PR, easier to track.